README.md

```
@@ -1,369 +1,45 @@
+```bash
+cargo test --workspace
+npm test
+```
+
+## CI Status
+
+All workflows green on main. See [Actions](https://github.com/Labpics-Team/lab-colors/actions) for live status.
+
+## License
+
+MIT
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)